server/authentication/authInit.js

```diff
@@ -27,7 +27,7 @@ module.exports = (userService, audit) => {
       const user = await getUser(accessToken, refreshToken, params.expires_in, params.user_name)
       return done(null, user)
     } catch (error) {
-      logger.error('Sign in error ', error.stack)
+      logger.error(`Sign in error for user: '${params.user_name}'`, error.stack)
```

Comment:
Adding user name to log out put to allow detecting which user tries to access licences without having the correct roles.